tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/cluster/bridge/BridgeTest.java

```diff
@@ -554,6 +559,44 @@ public void testClientSessionFactoryLeak() throws Exception {
       assertTrue(serverLocator.getClientSessionFactoryCount() <= 1);
    }
 
+   @TestTemplate
+   public void testBridgeClientID() throws Exception {
+      final String clientId = RandomUtil.randomString();
+      Map<String, Object> server0Params = new HashMap<>();
+      server0 = createClusteredServerWithParams(isNetty(), 0, true, server0Params);
+
+      Map<String, Object> server1Params = new HashMap<>();
+      addTargetParameters(server1Params);
+      server1 = createClusteredServerWithParams(isNetty(), 1, true, server1Params);
+
+      final String testAddress = "testAddress";
+      final String queueName0 = "queue0";
+      final String forwardAddress = "forwardAddress";
+      final String queueName1 = "queue1";
+
+      TransportConfiguration server1tc = new TransportConfiguration(getConnector(), server1Params);
+
+      HashMap<String, TransportConfiguration> connectors = new HashMap<>();
+      connectors.put(server1tc.getName(), server1tc);
+      server0.getConfiguration().setConnectorConfigurations(connectors);
+
+      List<String> connectorConfig = List.of(server1tc.getName());
+      // intentionally configure a bridge that will fail to connect and attempt to reconnect multiple times quickly
+      server0.getConfiguration().setBridgeConfigurations(List.of(new BridgeConfiguration().setName("bridge1").setQueueName(queueName0).setForwardingAddress(forwardAddress).setRetryInterval(0).setReconnectAttempts(-1).setStaticConnectors(connectorConfig).setClientId(clientId)));
```

Comment:
We should avoid setting infinite reconnects in tests, it just makes it more likely for bad things to happen later, e.g. cross-test contamination when something unexpected isnt handled nicely.

The test only seems to allow 2 seconds for connection, so the attempts + intervals only need to span that.

(I'd also name the bridge based on the test name, helps identify them in logs later in debug, and also useful for finding contaminations)